server.js

```diff
@@ -257,6 +257,12 @@ app.use('/pl/administrator/overview', require('./pages/administratorOverview/adm
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
+// Webhooks //////////////////////////////////////////////////////////
+app.use('/pl/webhooks/autograder', require('./webhooks/autograder/autograder'));
```

Comment:
At the moment this placement of the route is only working because we are running locally in dev mode. The problem is the `app.use(require('./middlewares/authn'));` line that in dev mode just automatically authenticates, but in production will fail.

A hacky solution would be to modify `middlewares/authn.js` to just check for the `/pl/webhooks/` path and succeed early with no `authn_user` set. This might actually be the best solution, despite being hacky. An alternative would be to only load `middlewares/authn` for those paths that need it, so in `server.js` we would have:

```javascript
app.use('/pl/course_instance', require('./middlewares/authn'));
app.use('/pl/course', require('./middlewares/authn'));
```

This feels a bit fragile because we'd need to remember to list all prefixes that need authn, and not accidentally miss any.

We could also do the check for `/pl/webooks/` in `server.js` like:

```javascript
app.use('^/pl(?!/webooks)', require('./middlewares/authn'));
```

I'm not sure whether this is better or worse than just hacking the `/pl/webooks/` special case into `authn.js`.